laughter/laughter_classification/rnn_laugh_classifier.py

```diff
@@ -0,0 +1,99 @@
+from timeit import default_timer as timer
+
+import torch
+import torch.nn as nn
+
+
+class RnnLaughClassifier(nn.Module):
+    def __init__(self, input_size1, input_size2, hidden_size1, hidden_size2):
+        super(RnnLaughClassifier, self).__init__()
+
+        self.input_size1 = input_size1
+        self.hidden_size1 = hidden_size1
+        self.lstm1 = nn.LSTM(input_size1, hidden_size1)
+        self.fc1 = nn.Linear(hidden_size1, 2)
+
+        self.input_size2 = input_size2
+        self.hidden_size2 = hidden_size2
+        self.lstm2 = nn.LSTM(input_size2, hidden_size2)
+        self.fc2 = nn.Linear(hidden_size1 + hidden_size2, 2)
+
+        self.init_hidden()
+
+    def init_hidden(self, cuda=False):
+        if cuda:
+            self.hidden1 = (
+                torch.zeros(1, 1, self.hidden_size1).cuda(),
```

Comment:
You can use `.to(device)` instead of this if statement

Reply:
Fixed